arroyo-api/src/sources.rs

```diff
@@ -373,13 +385,14 @@ pub struct SourceSchema {
 
     // set to true to use the kafka schema serialization wire format
     // (https://docs.confluent.io/platform/current/schema-registry/serdes-develop/index.html#wire-format)
-    kafka_schema: bool,
+    kafka_serialization_mode: Option<KafkaSerializationMode>,
 }
 
 fn builtin_for_name(name: &str) -> Result<SourceSchema, String> {
     match name {
         "nexmark" => Ok(nexmark_schema()),
         "impulse" => Ok(impulse_schema()),
+        "raw_kafka" => Ok(raw_kafka_schema()),
```

Comment:
Should this be something more generic than "raw_kafka"? It seems like any other stream source could similarly have a raw format.

Separately, should this be a builtin or a separate SourceSchema? The existing builtins are used solely for demo sources that have particular schemas.